rust/python-bindings/src/lib.rs

```
@@ -0,0 +1,71 @@
+use pyo3::PyResult;
+use pyo3::pymodule;
+use pyo3::types::PyModule;
+
+mod passwords;
+mod schema_store;
+mod validation;
+
+#[pymodule]
+#[pyo3(name = "_bindings")]
+pub mod bindings {
+    use super::*;
```

Clippy (1.96.0)

[error] usage of wildcard import

(error)